README.adoc

```diff
@@ -36,79 +35,94 @@ PowerPC::            NetBSD
 ARM::                NetBSD
 SPARC::              Solaris, Linux, NetBSD
 
-Other operating systems for the processors above have not been tested,
-but the compiler may work under other operating systems with little work.
+Other operating systems for the processors above have not been tested, but
+the compiler may work under other operating systems with little work.
 
-Before the introduction of objects, OCaml was known as Caml Special
-Light. OCaml is almost upwards compatible with Caml Special Light,
-except for a few additional reserved keywords that have forced some
-renaming of standard library functions.
+Before the introduction of objects, OCaml was known as Caml Special Light.
+OCaml is almost upwards compatible with Caml Special Light, except for a few
+additional reserved keywords that have forced some renaming of standard
+library functions.
 
-== CONTENTS
+== Contents
 
+  appveyor.yml::          configuration for appveyor CI
+  appveyor_build.sh::     appveyor CI script
```

Comment:
I'm not sure we should list CI files. They are not very important for users and people who cares will know what to look for. Also, you would need to list several more files (`.travis*`).